openlibrary/plugins/worksearch/schemes/authors.py

```diff
@@ -8,39 +7,41 @@
 
 
 class AuthorSearchScheme(SearchScheme):
-    universe = ['type:author']
-    all_fields = {
-        'key',
-        'name',
-        'alternate_names',
-        'birth_date',
-        'death_date',
-        'date',
-        'top_subjects',
-        'work_count',
-    }
-    non_solr_fields: set[str] = set()
-    facet_fields: set[str] = set()
-    field_name_map: dict[str, str] = {}
-    sorts = {
-        'work_count desc': 'work_count desc',
-        # Random
-        'random': 'random_1 asc',
-        'random asc': 'random_1 asc',
-        'random desc': 'random_1 desc',
-        'random.hourly': lambda: f'random_{datetime.now():%Y%m%dT%H} asc',
-        'random.daily': lambda: f'random_{datetime.now():%Y%m%d} asc',
-    }
-    default_fetched_fields = {
-        'key',
-        'name',
-        'birth_date',
-        'death_date',
-        'date',
-        'top_subjects',
-        'work_count',
-    }
-    facet_rewrites: dict[tuple[str, str], str | Callable[[], str]] = {}
+    def __init__(self):
+        super().__init__()
+        self._universe = {'type:author'}
```

Comment:
Also lets change this back to a list
```suggestion
        self.universe = ['type:author']
```